package.json

```diff
@@ -105,11 +105,10 @@
     "@patternfly/react-icons": "^3.7.5",
     "@redhat-cloud-services/entitlements-client": "^1.0.4",
     "@redhat-cloud-services/frontend-components": "0.0.5",
-    "@redhat-cloud-services/frontend-components-inventory": "0.0.11",
+    "@redhat-cloud-services/frontend-components-inventory": "0.0.12",
     "@redhat-cloud-services/frontend-components-remediations": "0.0.3",
     "@redhat-cloud-services/frontend-components-utilities": "0.0.5",
-    "@sentry/browser": "^5.3.0",
```

Comment:
Why is this dep removed? I don't think you should update any of these dependencies in this PR.

Reply:
Yeah this dep needs to be added back so Sentry works correctly.

Reply:
I have no idea how that got changed? I assume at some point I did something that updated dependencies but I don't know how sentry got removed. I went ahead and added it back but should I also go in and change all the other dependency versions back to what they were before they were updated?

Reply:
Yeah I would change them back to how they were, and then `npm install` again just so we can keep the scope of this PR limited to jwt auth stuff and not a dep update.

Reply:
got it